### DISCUSSION TITLE
Stronger warning message (displays related dashboards) when deleting a data source

### DISCUSSION THREAD

**Discussion by kenelm-ng** [grafana/grafana#62736] [Platform]
When deleting a data source, there is only a confirmation box saying "Are you sure you want to delete the "XYZ" data source?".
To avoid users delete data sources and invalidate dashboards,  a stronger warning message can be display in the confirmation box listing all the dashboards currently using the data source.
This should further lower the chance of users accidentally delete a data source.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82818. Feel free to continue the discussion around this there. Thank you!
